packages/terra-list/src/ListItem.jsx

```diff
@@ -75,6 +81,7 @@ const ListItem = ({
   onMouseDown,
   onSelect,
   refCallback,
+  isDisabled,
   ...customProps
 }) => {
   const listItemClassNames = cx([
```

Comment:
We'll want to ensure we don't display a disabled element as having interaction from hover styles as well.
```diff
- { 'is-selectable': isSelectable },
+ { 'is-selectable': !isDisabled && isSelectable },
```

Reply:
Changed 1c0de15